Mathlib/Algebra/Category/ModuleCat/Monoidal/Basic.lean

```diff
@@ -363,6 +363,9 @@ theorem hom_inv_associator {M N K : ModuleCat.{u} R} :
 
 namespace MonoidalCategory
 
+@[deprecated (since := "2025-10-29")] alias tensorObj := tensorObj_carrier
+@[deprecated (since := "2025-10-29")] alias tensorObj_def := tensorObj_carrier
```

Comment:
Added these aliases so that these [proofs](https://github.com/leanprover-community/mathlib4/pull/30638/files#diff-7799f55e2d9f10a2f813c5d5f3c35adf3de86799860e8836e3e1a52ab613c595L730-R727) in #30638 won't break and people know what to replace.